dist/debian/control

```diff
@@ -35,6 +36,7 @@ Depends:
  uidmap,
  squashfs-tools,
  fakeroot
+Recommends: libsubid4
```

Comment:
I would think this shouldn't be necessary because it should get automatically calculated at build time.  At least I know that's the way rpm works, although I'm not sure about deb/apt.

Reply:
All set, removed.